seacatauth/authn/service.py

```diff
@@ -367,12 +368,18 @@ async def m2m_login(
 		Direct authentication for M2M access (without login sessions)
 		This is NOT OpenIDConnect/OAuth2 compliant!
 		"""
+		# TODO: Where to get the tenant from
+		#   - in request? (header?)
+		#   - in M2M credentials? (this implies single-tenant credentials)
+		tenant = None
```

Comment:
How to handle M2M requests?
Should the tenant be included in request header?
Should the API keys/M2M credentials be single-tenanted?